ctsit/metab_import
# Add metab admin form to add/remove aliases.

**What does this change do?** _Please be clear and concise._

Add a form which allows you to create a new alias or delete an existing
alias. Also changed the names table to extend the unique constraint to a withheld
boolean as well. This means that when you mark someone withheld you can still use
their name as an alias. Also update change user's withheld to mark all names as the same
updated withheld status.

**Why was this change made?** _Including an issue number is sufficient. Otherwise, briefly explain the benefit of the change._

Completes SEP-133


## Verification

_List the steps needed to make sure this thing works. Be precise._

1. Setup a local supplemental database by following the steps in the README.
1. Start a local metab_admin instance with the steps in the README
1. Fill out the alias form
1. Verify that the data in the database match (names withholding changes from true/false, names get added).

- Verify the thing does this:
    - Ensures that active aliases are unique
- Pictures:

![image](https://user-images.githubusercontent.com/3639154/67789941-3a5a5300-fa4b-11e9-834e-313a5d623d2f.png)

## Affirmations

All of these should have a check by them. Any exception requires an explanation.
